tests/Java.Interop.Tools.Maven-Tests/Extensions/TestDataExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using System.Xml.Linq;
+using Java.Interop.Tools.Maven.Models;
+
+namespace Java.Interop.Tools.Maven_Tests.Extensions;
+
+static class TestDataExtensions
+{
+	public static Project CreateProject (Artifact artifact, Project? parent = null)
```

Comment:
Given that this is within an `*Extensions` class, I'm surprised this isn't an extension method.

Reply:
Meh, it's a test assembly and I didn't want an entire class for it.  😁